combat-trainer.lic

```diff
@@ -1979,6 +1982,11 @@ class GameState
     weapon_skill == 'Brawling'
   end
 
+  def brawl_tactics?
+    return true if !brawling?
+    ((weapon_skill == 'Brawling') && (weapon_name == ''))
```

Comment:
If the weapon skill is anything *but* Brawling, line 2026 will return. Therefore we know the first clause of line 2027 will always be true.